docs/AboutUs.md

```diff
@@ -4,49 +4,36 @@ We are a team based in the [School of Computing, National University of Singapor
 
 ## Project Team
 
-#### [Damith C. Rajapakse](http://www.comp.nus.edu.sg/~damithch) <br>
-<img src="images/DamithRajapakse.jpg" width="150"><br>
+#### [Chuan Wei Candiie](https://github.com/Candiie) <br>
+<img src="images/Candiie.png", width="150"><br>
 **Role**: Project Advisor
 
------
+----- 
 
-#### [Joshua Lee](http://github.com/lejolly)
-<img src="images/JoshuaLee.jpg" width="150"><br>
-Role: Developer <br>  
-Responsibilities: UI
+#### [Lee Wenwei Johnervan](http://github.com/johnervan) <br>
+<img src="images/JohnervanLee.png", width="150"><br>
+**Role**: Team Lead <br>
+Responsibilities: Documentation, Deliverables & Deadlines, Scheduling & Tracking, Model
 
 -----
 
-#### [Leow Yijin](http://github.com/yijinl) 
-<img src="images/LeowYijin.jpg" width="150"><br>
-Role: Developer <br>  
-Responsibilities: Data
+#### [Calvin Yang Jiawei](http://github.com/origiri) <br>
+<img src="images/CalvinYang.png", width="150"><br>
+**Role**: Developer <br>
+Responsibilities: Integration, Storage
```

Comment:
@weikangchia @jaeoheeail @johnervan @origiri  responsibilities can be further defined/elaborated *hint* take a look at this ~ https://github.com/CS2103AUG2016-F11-C2/main/blob/7a5b70436f2f45f65a0156cb8ebae6ddc7d94791/docs/AboutUs.md